src/library/processInlineTag.js

```diff
@@ -32,7 +32,7 @@ export default function processInlineTag(
         style.add(`bgcolor-${backgroundColor.replace(/ /g, '')}`);
       }
       if (fontSize) {
-        style.add(`fontsize-${fontSize.substr(0, fontSize.length - 2)}`);
+        style.add(`fontsize-${fontSize.replace(/px$/g, '')}`);
       }
```

Comment:
Change looks good I am just thinking why not trim and then remove last 2 chars.

Reply:
I'm not quite sure what you mean by 'trim'. The issue with removing the last two characters is if `fontSize` is `inherit`, or `medium`, in which case you don't want to lose the last two chars.

Reply:
Ah I get you point - I was thinking only about px values, thanks.